WPMedia/fusion-news-theme-blocks
# THEMES-1520: Enable testing (and linting THEMES-1460)

## Description

Enables testing and partially linting

## Jira Ticket

- [THEMES-1520](https://arcpublishing.atlassian.net/browse/THEMES-1520)

## Test Steps

1. Checkout this branch `git checkout themes-1520`
2. Run fusion repo with linked blocks `npm run test -- --coverage`
3. Ensure tests run successfully with coverage adjusted

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1520]: https://arcpublishing.atlassian.net/browse/THEMES-1520?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
